Header header
geometry_msgs/Point[4] contours
uint8 state
uint8 EMPTY=0
uint8 BLUE=1
uint8 RED=2
uint8 UNDEFINED=3